lib/rules/no-standalone-backslash.ts

```diff
@@ -0,0 +1,45 @@
+import type { RegExpVisitor } from "regexpp/visitor"
+import type { RegExpContext } from "../utils"
+import { createRule, defineRegexpVisitor, CP_BACK_SLASH } from "../utils"
+
+export default createRule("no-standalone-backslash", {
+    meta: {
+        docs: {
+            description: "disallow standalone backslash (`\\`)",
```

Comment:
It has to be plural here.

```suggestion
            description: "disallow standalone backslashes (`\\`)",
```

Reply:
Thank you!